apps/vmq_proto/include/on_client_gone_pb.hrl

```diff
@@ -31,6 +31,12 @@
             | 'REASON_MQTT_CLIENT_DISCONNECT'
             | 'REASON_RECEIVE_MAX_EXCEEDED'
             | 'REASON_PROTOCOL_ERROR'
+            | 'REASON_PUBLISH_AUTH_ERROR'
+            | 'REASON_INVALID_PUBREC_ERROR'
+            | 'REASON_INVALID_PUBCOMP_ERROR'
+            | 'REASON_UNEXPECTED_FRAME_TYPE'
+            | 'REASON_EXIT_SIGNAL_RECIVED'
```

Comment:
RECEIVED -> typo

Reply:
Done

Reply:
Where is this done? I can't see in the changes.

Reply:
It should be there now.